### PR TITLE
Adding ORCA 6.0.0, Gaussian-16.C.02-AVX2, VASP5.4.4, FHI-aims, PyMOL, and VMD eb files

### PR DESCRIPTION
**Pull request by geoffreyweal**
This pull request is to allow installation of ORCA 6.0. This requires the updated orca.py file in easyblocks (https://github.com/easybuilders/easybuild-easyblocks/blob/9bf81c21cedc1d77f78409c1423889ebe8a2694a/easybuild/easyblocks/o/orca.py) for ORCA 6.0 to install.

- https://github.com/easybuilders/easybuild-easyblocks/pull/3395

### CODE REVIEW

**Review by cperezmig-dipc — APPROVED** [review]

File: easybuild/easyconfigs/o/ORCA/ORCA-6.0.0-avx2-gompi-2023b.eb [hunk 1]
```diff
@@ -0,0 +1,23 @@
+name = 'ORCA'
+version = '6.0.0-avx2'
```
Comment (smoors) [inline easybuild/easyconfigs/o/ORCA/ORCA-6.0.0-avx2-gompi-2023b.eb:2]: `-avx2` is better suited as a `versionsuffix`; the version is still 6.0.0

Comment (geoffreyweal) [reply easybuild/easyconfigs/o/ORCA/ORCA-6.0.0-avx2-gompi-2023b.eb]: Excellent. I have done this now and recommitted

Comment (smoors) [reply easybuild/easyconfigs/o/ORCA/ORCA-6.0.0-avx2-gompi-2023b.eb]: thanks a lot. please also rename the file accordingly to `ORCA-6.0.0-gompi-2023b-avx2.eb`

**Review by geoffreyweal — COMMENTED** [review]
Changes made as requested

**Comment by geoffreyweal** [comment]
I have also included here a eb file for uploading Gaussian that have been waiting to be processed for some time, but then I needed a new project to update the ORCA files. So I have put them both here and will change the title

**Comment by smoors** [comment]
@geoffreyweal i don't have access to Gaussian 16C at the moment, so i cannot test it.

can you submit a test report? otherwise better to move it into a separate PR

**Comment by geoffreyweal** [comment]
Cool I will do that at some point

**Comment by geoffreyweal** [comment]
I have been doin some other work and have updated some eb files to new versions of WMD, PyMOL. Apologies this is on the same cpull. I'm not versed with GitHub so this is the only way I know how to confirm I've made changes I have happy with. I probsbly should have just opened new branches from the original, but I didn't know what i was doing and how to do this. So apologies I have a lot of things on my plate atm.

**Comment by geoffreyweal** [comment]
> test report

I was wondering is this what I should do for submitting a test report? https://tutorial.easybuild.io/2020-06-isc20/contributing/

**Comment by geoffreyweal** [comment]
Im not sure how to submit a report test for EasyBuild, so i've quoted the output from what I did. 

First: This is the message I get if I add the Gaussian file to ```easybuild/source/g/Gaussian``` without changing its filename first

```
[admin@rockygpu ~]$ eb --robot Gaussian-16.C.02-AVX2.eb
== Temporary log file in case of crash /tmp/eb-t342zutu/easybuild-chh_ua5j.log
== resolving dependencies ...
== processing EasyBuild easyconfig /opt/ohpc/pub/libs/easybuild/4.9.1/easybuild/easyconfigs/g/Gaussian/Gaussian-16.C.02-AVX2.eb
== building and installing Gaussian/16.C.02-AVX2...
== fetching files...

Download instructions:


    Gaussian is licensed software. You need to obtain a copy of Gaussian 16.C.02-AVX2. 
    Once you have the DVD from Gaussian that contains the Gaussian installation files on it, 
    copy the E6B-432N.tbJ file from the Gaussian DVD (GaussianDVD/tar/E6B-432N.tbJ) 
    into the Gaussian source folder in EasyBuild (PATH_TO_EASYBUILD_FOLDER/sources/g/Gaussian).
    Make sure to change the name of the tbJ file in the sources folder to 16.C.02-AVX2-E6B-432N.tbJ


== FAILED: Installation ended unsuccessfully (build directory: /home/admin/easybuild/build/Gaussian/16.C.02/system-system-AVX2): build failed (first 9999999999999999999 chars): Couldn't find file 16.C.02-AVX2-E6B-432N.tbJ anywhere, please follow the download instructions above, and make the file available in the active source path (/home/admin/easybuild/sources) (took 0 secs)
== Results of the build can be found in the log file(s) /tmp/eb-t342zutu/easybuild-Gaussian-16.C.02-20240819.185209.tECgw.log
ERROR: Build of /opt/ohpc/pub/libs/easybuild/4.9.1/easybuild/easyconfigs/g/Gaussian/Gaussian-16.C.02-AVX2.eb failed (err: "build failed (first 9999999999999999999 chars): Couldn't find file 16.C.02-AVX2-E6B-432N.tbJ anywhere, please follow the download instructions above, and make the file available in the active source path (/home/admin/easybuild/sources)")
```

Second: Once I change the name of the file to that given, this is what I get:

```
[admin@rockygpu Gaussian]$ eb --robot Gaussian-16.C.02-AVX2.eb
== Temporary log file in case of crash /tmp/eb-ovar3jxn/easybuild-gixrgafp.log
== resolving dependencies ...
== processing EasyBuild easyconfig /opt/ohpc/pub/libs/easybuild/4.9.1/easybuild/easyconfigs/g/Gaussian/Gaussian-16.C.02-AVX2.eb
== building and installing Gaussian/16.C.02-AVX2...
== fetching files...
== ... (took 8 secs)
== creating build dir, resetting environment...
== unpacking...
== ... (took 3 mins 26 secs)
== patching...
== preparing...
== configuring...
== building...
== testing...
== installing...
== ... (took 18 secs)
== taking care of extensions...
== restore after iterating...
== postprocessing...
== sanity checking...
== ... (took 1 secs)
== cleaning up...
== creating module...
== permissions...
== packaging...
== COMPLETED: Installation ended successfully (took 3 mins 54 secs)
== Results of the build can be found in the log file(s) /home/admin/easybuild/software/Gaussian/16.C.02-AVX2/easybuild/easybuild-Gaussian-16.C.02-20240819.185739.log
== Build succeeded for 1 out of 1
== Temporary log file(s) /tmp/eb-ovar3jxn/easybuild-gixrgafp.log* have been removed.
== Temporary directory /tmp/eb-ovar3jxn has been removed.
```

**Comment by geoffreyweal** [comment]
@boegelbot please test

**Comment by geoffreyweal** [comment]
@boegelbot please test @ generoso

**Comment by smoors** [comment]
@geoffreyweal huge PRs like this are a pain to review and test.
please split it up into smaller PRs, preferably only keeping easyconfigs that belong together or are dependencies of each other.
for straightforward updates, it's ok to bundle multiple easyconfigs in 1 PR up to let's say 10 (but less is definitely better).
for (commercial) software that needs a manual download it's better to make a separate PR.

the simplest and recommended way that almost everyone uses is with the github integration built into EasyBuild, see here for the docs: https://docs.easybuild.io/integration-with-github/?h=github#integration_with_github

if you have issues getting it to work (or any other issues), feel free to ask in the EB slack: https://easybuild.io/join-slack

**Comment by geoffreyweal** [comment]
Hey @smoors Thanks for this. I still quite new to how to use Github in a collaboration and couldn't figure out how to push new scripts without them smothering them into one big push. I'll Close this push and try using your guide to make individual PRs. Thanks heaps for your patiance on this one.

**Comment by smoors** [comment]
no problem!, we're really happy with your contributions!

and again, don't hesitate to ask for help, we're a friendly community (but mostly volunteers).

**Pull request closed without merging**